test/index.test.js

```diff
@@ -104,6 +104,17 @@ describe('Elevio', function() {
         analytics.assert(window._elev.user.plan instanceof Array);
         analytics.assert(window._elev.user.plan[0] === 'gold');
       });
+
+      it('should send their traits', function() {
+        analytics.identify('id', { locale: 'en_US' });
+        analytics.assert(window._elev.user.traits instanceof Object);
+        analytics.assert(window._elev.user.traits.locale === 'en_US');
+      });
+
+      it('should not send traits', function() {
```

Comment:
you should also add separate tests asserting that reserved keys are omitted. this case in particular is not currently being tested:

```js
analytics.identify('id', {
  firstName: 'Test', // reserved
  locale: 'en_US'    // not reserved
});
```